packages/data/lib/di/data_injector.dart

```diff
@@ -40,6 +44,27 @@ Future<void> _initTractApiService() async {
   );
 }
 
+Future<void> _initTMDBApiService() async {
+  final appConfigService = GetIt.I.get<AppConfigService>();
+
+  final tmdbApiKey = await appConfigService
+      .getConfigValue<String>(TMDBConfig.tmdbApiKeyJsonConfigName);
+
+  GetIt.I.registerSingleton<Dio>(
+    _buildDioForTMDBApi(tmdbApiKey),
+    instanceName: DISingletonInstanceNames.tmdbApiDio,
+  );
+
+  GetIt.I.registerSingleton<ApiBaseService>(
+    ApiServiceImpl(
+      GetIt.I.get<Dio>(
+        instanceName: DISingletonInstanceNames.tmdbApiDio,
+      ),
+    ),
+    instanceName: DISingletonInstanceNames.tmdbApiService,
+  );
```

Comment:
@sergei-mikhailovskii-idf Does it make sense to have several dio and apiService instances when the only difference is base url?